src/views/image/TemplateZones.vue

```diff
@@ -32,15 +32,15 @@
       <template slot="action" slot-scope="text, record">
         <span style="margin-right: 5px">
           <a-button
-            :disabled="!('copyTemplate' in $store.getters.apis)"
+            :disabled="!('copyTemplate' in $store.getters.apis && record.isready)"
             icon="copy"
             shape="circle"
             :loading="copyLoading"
             @click="showCopyTemplate(record)" />
         </span>
         <span style="margin-right: 5px">
           <a-button
-            :disabled="!('deleteTemplate' in $store.getters.apis)"
+            :disabled="!('deleteTemplate' in $store.getters.apis && record.isready)"
```

Comment:
@Pearl1594 can you check against old UI, if deleting should be allowed? For example for error on download or uploading of local file?

Reply:
ping @Pearl1594

Reply:
@rhtyd the legacy UI allows deletion of templates / ISOs incase there is a failure in downloading it